runtime/doc/lua.txt

```diff
@@ -1415,14 +1415,14 @@ schedule_wrap({cb})                                      *vim.schedule_wrap()*
 deep_equal({a}, {b})                                        *vim.deep_equal()*
                 Deep compare values for equality
 
-                Tables are compared recursively unless they both provide the `eq` methamethod. All other types are compared using the equality `==` operator.
+                Tables are compared recursively unless they both provide the `eq` metamethod. All other types are compared using the equality `==` operator.
```

Comment:
Hmm why is this line not wrapped?

Reply:
Hmm, good question. I can definitely wrap that one.

While I'm at it, should I also maximize line length within the `78` char limit where it doesn't hurt readability, i.e. in the middle of a regular sentence? There are a few paragraphs that wrap earlier than needed.

Reply:
This is a quirk with the generator, doesn't need to be solved here.